src/main/java/com/google/devtools/build/lib/remote/GrpcRemoteCache.java

```diff
@@ -268,7 +269,7 @@ public void ensureInputsPresent(
     if (!options.remoteInstanceName.isEmpty()) {
       resourceName += options.remoteInstanceName + "/";
     }
-    resourceName += "blobs/" + digestUtil.toString(digest);
+    resourceName += "blobs/" + digest.getHash() + "/" + digest.getSizeBytes();
```

Comment:
```suggestion
    resourceName += "blobs/" + DigestUtil.toString(digest);
```

Reply:
This doesn't work due to breakages due to dependency issues. Seems like a small price to pay for now.